code/api/src/migrations/20210208182658-add_styleType_to_products.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.addColumn(
+      'products',
+      'styleType',
```

Comment:
styleType not to be confused with 'type' which refers to the clothing type ie cloth, accessories.